### DISCUSSION TITLE
Problems to get cookies from selenium session and apply it in `httpx.Client()`

### DISCUSSION THREAD

**Discussion by danbailo** [projectdiscovery/httpx#937] [Q&A] [answered]
I'm not getting to get selenium session cookies and instantiate an object(`httpx.Client()`) with this cookies. I tried a lot and it's does not works :( (the requests does not return the expected response, such as OK status). Does anyone tell me how to do it?

**Comment by olearycrew** [comment, marked as answer]
The HTTP based client we use allows for a "Jar" that specifies cookies for the request

<img width="709" alt="CleanShot 2023-04-19 at 11 55 11@2x" src="https://user-images.githubusercontent.com/6044920/233132281-d4ceb3a2-6d8d-4146-a049-c311ffd9e853.png">
